static/app/gettingStartedDocs/dotnet/aspnet.tsx

```diff
@@ -178,9 +183,51 @@ const onboarding: OnboardingConfig = {
   ],
 };
 
+const crashReportOnboarding: OnboardingConfig = {
+  introduction: () => getCrashReportModalIntroduction(),
+  install: (params: Params) => [
+    {
+      type: StepType.INSTALL,
+      configurations: [
+        getCrashReportSDKInstallFirstStep(params),
+        {
+          description: tct(
+            'If you are rendering the page from the server, for example on ASP.NET MVC, the [code:Error.cshtml] razor page can be:',
+            {code: <code />}
+          ),
+          code: [
+            {
+              label: 'Cshtml',
```

Comment:
```suggestion
              label: 'cshtml',
```

Reply:
should probably change this in the docs too then👀 https://docs.sentry.io/platforms/dotnet/guides/aspnet/user-feedback/
<img width="573" alt="SCR-20240308-kecn" src="https://github.com/getsentry/sentry/assets/56095982/70c11fd3-0368-4442-81bb-c24895835451">

Reply:
ah i think docs automatically capitalizes the label name actually :(

 https://github.com/getsentry/sentry-docs/blob/4e9b1756ccbdeb1aee1ab5fa69646e53ed305b18/platform-includes/user-feedback/example-widget/dotnet.aspnetcore.mdx#L13